packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -117,155 +117,156 @@ const IDVForm = ({
+                                                            generatePlaceholderText(selected_doc)
+                                                        }
+                                                    />
+                                                )}
+                                            </Field>
+                                        </fieldset>
+                                        {values.document_type.additional?.display_name && (
+                                            <fieldset className='additional-field'>
+                                                <Field name='document_additional'>
+                                                    {({ field }: FieldProps) => (
                                                         <Input
                                                             {...field}
-                                                            name='document_number'
+                                                            name='document_additional'
                                                             bottom_label={
-                                                                values.document_type &&
+                                                                values.document_type.additional &&
                                                                 getExampleFormat(
-                                                                    values.document_type.example_format ?? ''
+                                                                    values.document_type.additional?.example_format
                                                                 )
                                                             }
                                                             disabled={!values.document_type.id}
                                                             error={
-                                                                (values.document_type.id &&
-                                                                    touched.document_number &&
-                                                                    errors.document_number) ||
+                                                                (touched.document_additional &&
+                                                                    errors.document_additional) ||
                                                                 errors.error_message
                                                             }
                                                             autoComplete='off'
-                                                            placeholder={generatePlaceholderText(selected_doc)}
-                                                            value={values.document_number}
+                                                            placeholder={`Enter your ${values.document_type.additional?.display_name.toLowerCase()}`}
```

Comment:
How does this handle the translation?

Reply:
Thank you for pointing that. It was never localized from the start. Will try to add that in this PR